src/navigation/routeKeys.ts

```diff
@@ -12,3 +14,21 @@ export const myIdentity = 'MyIdentity' as const;
 export const councilScreen = 'CouncilScreen' as const;
 export const treasuryScreen = 'TreasuryScreen' as const;
 export const motionsScreen = 'MotionsScreen' as const;
+
+export const linking: LinkingOptions = {
+  prefixes: ['litentry://'],
+
+  config: {
```

Comment:
What is this here?

Reply:
this is how you define a path for your screen. [more info](https://reactnavigation.org/docs/configuring-links#mapping-path-to-route-names).

so for example, this config says if I use `linkTo('treasury')` or I receive a deeplink with the format `litentry://treasury` the screen `treasuryScreen` should be opened.

it also make sure that the navigation state is fine. so the route history would be `['App', 'Dashboard', 'Treasury']`

Reply:
What I worry about is that this become "untested" code, when we rename `App` to `SomethingElse`, would that be the case?

Reply:
yes make sense, I've made 'App' as a route key so the variable would be the reference and any change to that would change the linking also.